src/tile-model-factory.ts

```diff
@@ -0,0 +1,136 @@
+import { TileModel, RoadType } from './tile-model'
+
+const makeTileModel = (id: number): TileModel => {
```

Comment:
should/could `id` be an `enum`?

Reply:
What would you name each case?

Reply:
true. I guess that would probably mean that we would need to name each tile

Reply:
Happy to make this an `enum` if we can think of a good naming schema.